src/Compilers/VisualBasic/Test/Emit/Emit/FieldInitializerTests.vb

```diff
@@ -583,6 +583,7 @@ BC30424: Constants must be of an intrinsic or enumerated type, not a class, stru
 </expected>)
         End Sub
 
+#If NET46 Then
```

Comment:
Is there some type here that causes the test to be unable to be compiled on coreclr? If not, why not use a ConditionalFact?